src/dmd/backend/debugprint.d

```diff
@@ -123,11 +123,10 @@ const(char)* class_str(SC c)
  *      pointer to string
  */
 
-@trusted
-const(char)* oper_str(uint oper)
+const(char)* oper_str(uint oper) pure
 {
     assert(oper < OPMAX);
-    return debtab[oper];
+    return &debtab[oper][0];
```

Comment:
Why is this needed? Also, why is the trusted omitted? It looks like this function could have been safe before this change.

Reply:
Doesn't hurt to add more bounds checking to code this brittle + friendlier to @safe rules